### DISCUSSION TITLE
Adding needs feature at a workflows scale

### DISCUSSION THREAD

**Discussion by hadialqattan** [community/community#15700] [Actions]
I'm wondering why didn't Github add a feature like [Relies-On](https://github.com/hadialqattan/relies-on) natively?

(Relies-On is a Github action to identify any workflows that must complete successfully before another workflow will run. It's more like <code><a href="https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idneeds">needs</a></code> at a workflows scale).

So then we will be able to do:
> cd.yml
```cd.yml
name: CD
needs: CI
...
```

**Comment by TWiStErRob** [comment]
Related https://github.community/t/depend-on-another-workflow/16311
